types/nationalAvalancheCenter/observations.ts

```diff
@@ -0,0 +1,244 @@
+const Unknown = 'Unknown';
+
+function reverseLookup<T extends PropertyKey, U extends PropertyKey>(mapping: Record<T, U>, lookup: U): string {
+  return Object.entries(mapping)
+    .filter(([_, value]) => value === lookup)
+    .map(([key, _]) => key)
+    .reduce((accumulator, currentValue) => currentValue, Unknown);
```

Comment:
is the `reduce` just to get the first matching value?

Reply:
Yep!